vehicle/src/main/scala/com/wedriveu/vehicle/boundary/VehicleConfiguratorView.scala

```diff
@@ -3,23 +3,37 @@ package com.wedriveu.vehicle.boundary
 import java.awt.event.{ActionEvent, ActionListener, WindowAdapter, WindowEvent}
 import javax.swing._
 
-import com.wedriveu.vehicle.entity.VehicleCreator
+import com.wedriveu.vehicle.control.VehicleCreator
 
 /**
-  * Created by Michele on 02/08/2017.
+  * @author Michele Donati 02/08/2017.
   */
-trait VehicleConfiguratorView {
 
+/** This trait models the UI of the vehicles configurator file.*/
+trait VehicleConfiguratorView {
```

Comment:
Please remember that also the speed has to be configurable. So the `speedTextField ` must be editable, and its value must be assigned as the speed of the vehicle.